src/CombinePDF/CombinePDF.tsx

```diff
@@ -110,6 +156,12 @@ const CombinePDF = (): JSX.Element => {
                     key={pageNumber}
                     className="shadow p-1 bg-white m-1 rounded-md overflow-hidden border-4 border-white hover:cursor-move"
                   >
+                    <div
+                      onClick={(e) => onDelete(pageNumber)}
+                      className="absolute rounded-md p-1 bg-red-600 text-white text-xs cursor-pointer"
+                    >
+                      Delete
+                    </div>
```

Comment:
Can I suggest you move this to a button with icon `x` for example and to be styled on the top right corner of the file? since we are using tailwind I think it should be easy.